src/lib/components/Nav.svelte

```diff
@@ -67,12 +75,13 @@
       {#each pages as page}
         <a
           class={classNames(
-            'rounded-md px-3 py-2 transition-all dark:shadow-sm dark:shadow-secondary-100',
+            'text-nav rounded-md px-3 py-2 transition-all dark:shadow-sm dark:shadow-secondary-100',
             hash === page.href
               ? 'bg-gray-200 dark:bg-secondary-200'
               : 'hover:bg-gray-100 dark:hover:bg-secondary-100'
           )}
           href={page.href}
+          on:click={event => scrollToSection(event, page.href)}
```

Comment:
This is bad for web accessibility because screen readers/other web users don't necessarily use click. Some people might always prefer keyboard and use tab + enter keys. This one is a bit more complex so I will change later, but putting `on:click` on an anchor tag is not good.